packages/back-end/src/routers/sdk-connection/sdk-connection.controller.ts

```diff
@@ -45,7 +46,16 @@ export const postSDKConnection = async (
     params.environment,
   ]);
 
-  const doc = await createSDKConnection({ ...params, organization: org.id });
+  let encryptPayload = false;
+  if (orgHasPremiumFeature(org, "encrypt-features-endpoint")) {
```

Comment:
I'm not 100% sure we want to put the check here. If a user had previously enabled encryption and then their licensing expired, this would break their SDK integrations if they tried to update the connection

Reply:
This is on the controller's `postSDKConnection` method. Are we calling that for updates, too? I would assume we would be calling `putSDKConnection` on the controller for the updates.

Reply:
good point! resolving